src/monodroid/CMakeLists.txt

```diff
@@ -163,16 +180,16 @@ else()
         include_directories("/usr/local/opt/mingw-zlib/usr/x86_64-w64-mingw32/include")
       endif()
     endif()
+
+    link_directories("${XA_LIB_TOP_DIR}/${ANDROID_ABI}")
   endif()
 
   if(DEFINED HOST_BUILD_NAME)
-    include_directories("../../bin/${CMAKE_BUILD_TYPE}/include/${HOST_BUILD_NAME}")
-    include_directories("../../bin/${CMAKE_BUILD_TYPE}/include/${HOST_BUILD_NAME}/eglib")
-
-    if (DEFINED CONFIGURATION)
-      include_directories("../../bin/${CONFIGURATION}/include/${HOST_BUILD_NAME}")
-      include_directories("../../bin/${CONFIGURATION}/include/${HOST_BUILD_NAME}/eglib")
-    endif()
+    link_directories("${XA_LIB_TOP_DIR}/${HOST_BUILD_NAME}")
+    include_directories("${DEFAULT_BIN_DIR}/include/${HOST_BUILD_NAME}")
+    include_directories("${DEFAULT_BIN_DIR}/include/${HOST_BUILD_NAME}/eglib")
+    include_directories("../../bin/${CONFIGURATION}/include/${HOST_BUILD_NAME}")
```

Comment:
Should header files be extracted into `$(topdir)/bin/Debug/include`. Or should `bin/BuildDebug` be used?  Though if the header files are part of the “mono bundle”, they’ll _need_ to be in the former and not the latter...

Reply:
I'd prefer to keep them where the Mono archive puts them.